common_types/src/column.rs

```diff
@@ -674,8 +834,8 @@ macro_rules! define_column_block {
 
 // Define column blocks, Null is defined explicitly in macro.
 define_column_block!(
-    Timestamp, Double, Float, Varbinary, String, UInt64, UInt32, UInt16, UInt8, Int64, Int32,
-    Int16, Int8, Boolean, Date, Time
+    Timestamp, Double, Float, Varbinary, UInt64, UInt32, UInt16, UInt8, Int64, Int32, Int16, Int8,
```

Comment:
Why remove String from this?

Reply:
Because we need  use different implementation depend on `is_dictionary` variable.